web/src/app/admin/AdminConfig.tsx

```diff
@@ -180,7 +180,7 @@ export default function AdminConfig(): JSX.Element {
           >
             <AccordionSummary
               aria-expanded={section === groupID}
-              aria-controls={`accordion-sect-${groupID}`}
+              aria-controls={`accordion-section-summary-${groupID}`}
```

Comment:
Doesn't this need to match the `id=` below?
```suggestion
              aria-controls={`accordion-section-details-${groupID}`}
```

Reply:
it looks like the ID below isn't even needed (and is throwing duplicate ID errors), because MUI automatically sets the `id` fields properly if you provide `aria-controls`:

![Screen Shot 2022-04-25 at 1 13 30 PM](https://user-images.githubusercontent.com/11381794/165167512-21bed838-f5dd-47b3-ae7a-0f60dccd428c.png)

![Screen Shot 2022-04-25 at 1 14 44 PM](https://user-images.githubusercontent.com/11381794/165167571-3c928024-dd86-4eec-983d-df92ff81496c.png)